runtime/libffi/preconf/oa/ffi.h

```diff
@@ -0,0 +1,487 @@
+/* -----------------------------------------------------------------*-C-*-
```

Comment:
is this handwritten code?  or generated from something?

Reply:
Auto generated by running ./configure on our build machines. It's a bit of a bogus solution, but that was the solution we can up with to get from to work nicely with UMA

Reply:
Is there an issue that describes why we took this approach and whether we intend to do something different in the future?  I'd like a doc trail we can follow to find the rationale behind what've done to make it easier to refactor in the future

Reply:
I believe the logic was predicated on the assumption that we would be unable to run configure on the build machine. Although I'm not sure that assumption still holds true.  I believe maybe it was a limitation of the old style DDR, and it would require the generated files to exist inside the source zip